server/src-lib/Hasura/GraphQL/Execute/LiveQuery/Poll.hs

```diff
@@ -64,7 +65,12 @@ data Subscriber
   , _sOnChangeCallback :: !OnChange
   }
 
-type OnChange = GQResponse -> IO ()
+data OnChangeMeta
+  = OnChangeMeta
+  { _ocmExecutionTime :: !(Maybe Double)
+  }
+
+type OnChange = GQResponse -> OnChangeMeta -> IO ()
```

Comment:
What does the name `OnChangeMeta` mean? AFAICT, it just has the execution time. Maybe its a good idea to add some comments on this ADT?

Reply:
Oh so I think this means metadata? Do you think it makes sense to call it `LiveQueryMetadata`?